### ISSUE TITLE
Guest Groups table has difficult-to-read transparent rows

### ISSUE THREAD

**Issue by SkylerRingtail** [MidwestFurryFandom/rams#338]
Every other row on the Guest Groups page is transparent. While I appreciate our raccoon mascot, he is very distracting when trying to read a table.

This page is accessed strangely by going to People -> Bands/Guests/MIVS (any of the three). The issue applies to all 4 tabs of the resulting page, but "All groups" is the most obvious case.

Worth noting, adding groups via this page executes a completely different process than going to People -> Groups.

![image](https://user-images.githubusercontent.com/28108495/62094240-08660480-b242-11e9-9c6d-81e7fdab395c.png)

**Comment by kitsuta** [comment]
Huh, does MFF plan on actually using the GuestGroup plugin? It's fairly bespoke but also relatively simple to customize.

For a quick overview of how this actually works, each option represents a different department that works with different types of guests (MIVS is the MAGFest Indie Videogame Showcase). An admin inputs a group for each guest that's accepted our invitation, then marks them as the appropriate "type" of group. This generates:
1. An "info" page that's used by admins to input different parameters for generating agreements for guest groups to sign (we have plans to overhaul this for Super 2020).
2. A checklist for each group to fill out that, e.g., asks them for biographical/promotional info, asks them to tell us how they want to sell merch, and lets them submit a "guest" panel.
3. Automated emails reminding guests to fill out their checklist, because no one ever fills out their dang checklist. :)

**Comment by jmdawson** [comment]
I don't believe any MFF departments are aware of the page or have any intent to use it.
